src/mono/wasm/templates/templates/browser/main.js

```diff
@@ -24,5 +24,5 @@ const exports = await getAssemblyExports(config.mainAssemblyName);
 const text = exports.MyClass.Greeting();
 console.log(text);
 
-document.getElementById("out").innerHTML = `${text}`;
-await runMainAndExit(config.mainAssemblyName, ["dotnet", "is", "great!"]);
\ No newline at end of file
+document.getElementById('out').innerHTML = text;
+await runMainAndExit(config.mainAssemblyName, ['dotnet', 'is', 'great!']);
```

Comment:
This clashes with `withApplicationArgumentsFromQuery`. Let's use `dotnet.run()` instead.